ReactCommon/react/renderer/components/view/AccessibilityPropsMapBuffer.h

```diff
@@ -23,6 +23,7 @@ constexpr MapBuffer::Key AP_ACCESSIBILITY_STATE = 6;
 constexpr MapBuffer::Key AP_ACCESSIBILITY_VALUE = 7;
 constexpr MapBuffer::Key AP_ACCESSIBLE = 8;
 constexpr MapBuffer::Key AP_IMPORTANT_FOR_ACCESSIBILITY = 19;
+constexpr MapBuffer::Key AP_ACCESSIBILITY_UNIT = 47;
```

Comment:
cc @rshest 

Is it ok to have an arbitrary number here as the MapBuffer key?

Reply:
@ryancat @rshest 
I choose 47 because [AP_IMPORTANT_FOR_ACCESSIBILITY](https://github.com/fabriziobertoglio1987/react-native/blob/110b191b14e3cb692bb6a33f0f129b4f0215f9a6/ReactCommon/react/renderer/components/view/AccessibilityPropsMapBuffer.h#L24-L25) ([commit](https://github.com/fabriziobertoglio1987/react-native/commit/110b191b14e3cb692bb6a33f0f129b4f0215f9a6#diff-ae95b467322df81b65ff80322470e54dddd29a937d7fa8f6da7c4c8e7bfea76bR25)) is mapped to 19, while [AP_ACCESSIBLE](https://github.com/fabriziobertoglio1987/react-native/blob/110b191b14e3cb692bb6a33f0f129b4f0215f9a6/ReactCommon/react/renderer/components/view/AccessibilityPropsMapBuffer.h#L24-L25) is mapped to 8 in AccessibilityPropsMapBuffer.h. 
The original keys in [viewPropsConversions.h](https://github.com/fabriziobertoglio1987/react-native/commit/110b191b14e3cb692bb6a33f0f129b4f0215f9a6#diff-467726044e55973142e6ed0fccdb22f2f0bdd05555003e37c8987b0963f4b635L23-L64) would range from 0 to 44. 
Two keys (`VP_BORDER_CURVES = 45` and `VP_FG_COLOR = 46`) were later added in [ReactMapPropBufSetter.kt](https://github.com/facebook/react-native/blob/110b191b14e3cb692bb6a33f0f129b4f0215f9a6/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactMapBufferPropSetter.kt#L65-L66) and in [ViewPropsMapBuffer](https://github.com/facebook/react-native/blob/110b191b14e3cb692bb6a33f0f129b4f0215f9a6/ReactCommon/react/renderer/components/view/ViewPropsMapBuffer.h#L47-L48).

Maybe it is not required to respect the order. I'm happy to try to change this key to a sequential number and tests the changes. Thanks a lot.

Reply:
> Is it ok to have an arbitrary number here as the MapBuffer key?

Yes, it's totally fine having arbitrary number for the key, as long as it's not duplicated with an existing one (and to reduce the chance for that to happen, having a more structured/consistent order could be preferable).